src/app/pages/home/contact/contact.component.html

```diff
@@ -1 +1,83 @@
-<div>Contact Page</div>
\ No newline at end of file
+<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col">
+            <h1 align="center">
+                Contact <font color="#00BFFF">Us</font> <span class="blinking">_</span>
+            </h1>
+            <p align = "center">Being open source, being proactive and being welcome to every one.</p>
+            <p align = "center">With our slogan shown above, please feel free to contact us if you get any doubts.</p>
+        </div>
+    </div>
+
+    <div class="row">
```

Comment:
Consider using a `flex` box here instead. You can use Bootstrap's `d-flex` block.
```suggestion
    <div class="d-flex">
```